libraries/loggly_helpers.rb

```diff
@@ -0,0 +1,41 @@
+module LogglyHelpers
+  def configure_files(original_files)
+    valid_files(original_files).map(&:to_h).map do |file|
```

Comment:
@squaresurf I'm not sure why this .to_h is necessary, as we're already typechecking that we have a Hash.  I left it in just in case there's some reason for it, but if possible, I'd prefer to yank it.